modules/installation-osp-modifying-networktype.adoc

```diff
@@ -27,8 +26,8 @@ $ python -c '
 import yaml;
 path = "install-config.yaml";
 data = yaml.safe_load(open(path));
-data["networking"]["networkType"] = "Kuryr";
+data["networking"]["networkType"] = "OVN-K";
```

Comment:
Why the hell do we even have this module?! Anyway in the install-config.yaml the identifiers are like this:

OpenShiftSDN
OVNKubernetes

In this case we'd like to change to OpenShiftSDN (as OVNKubernetes is the default since 4.12 or 4.10), so the line has to read:

```
data["networking"]["networkType"] = "OpenShiftSDN";
```